src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -188,7 +201,7 @@ def __writeSub(self, executable, nJobs, location, processors):
         sub = """
 executable = %(executable)s
 universe = %(targetUniverse)s
-use_x509userproxy = true
+%(useCredentials)s
```

Comment:
```suggestion
use_x509userproxy = true
%(useCredentials)s
```
And remove from useCredentials

Reply:
Agreed